app/Platform/Concerns/BuildsGameListQueries.php

```diff
@@ -340,6 +340,7 @@ private function applyReleasedAtSorting(Builder $query, string $sortDirection =
                     ELSE 4
                 END AS granularity_order
             SQL)
+            ->orderByRaw('released_at IS NULL')
```

Comment:
```suggestion
            // Ensure NULL release dates always sort to the end, regardless of sort direction.
            ->orderByRaw('released_at IS NULL')
```
Recommend adding a comment explaining the "why", otherwise the intent of this sort may be lost.

Reply:
suggestion makes sense. Thanks